pkg/master/master.go

```diff
@@ -549,6 +549,46 @@ func (m *Master) InstallSwaggerAPI() {
 	swagger.RegisterSwaggerService(swaggerConfig, m.handlerContainer)
 }
 
+// Picks the best available address for an internal health check
+// TODO: Create utility function, combining this an GetHostIP in kubelet.go?
+func pickAddress(node *api.Node) string {
```

Comment:
#7182 does/should unbreak AWS for now - it would be good to get that merged!

AWS needs this and #7115 if I wanted to change the node name to the EC2 instance id, which would be great to do long-term.  In #7182 I put out the fire, so this can wait until #7092 is implemented.

I don't suppose there's a different reason to merge this and/or #7115? @cjcullen you mentioned you were also looking at this (for a different reason, I guess)